src/main/java/it/gov/pagopa/payhub/activities/connector/ionotification/mapper/NotificationRequestMapper.java

```diff
@@ -50,28 +52,41 @@ public List<NotificationRequestDTO> map(List<PaymentOptionDTO> paymentOptions, L
                         (i1, i2) -> i1
                 ))
                 .values().stream()
-                .map(installmentDTO -> mapNotificationRequestDTO(orgId, debtPositionTypeOrgId, ioNotificationDTO, installmentDTO, operationType))
+                .map(installmentDTO -> mapNotificationRequestDTO(orgId, debtPositionTypeOrgId, ioNotificationDTO, installmentDTO))
                 .toList();
     }
 
-    private static NotificationRequestDTO mapNotificationRequestDTO(
-            Long orgId, Long debtPositionTypeOrgId, IONotificationDTO ioNotificationDTO,
-            InstallmentDTO installmentDTO, NotificationRequestDTO.OperationTypeEnum operationType) {
+    private NotificationRequestDTO mapNotificationRequestDTO(Long orgId, Long debtPositionTypeOrgId, IONotificationDTO ioNotificationDTO,
+                                                                    InstallmentDTO installmentDTO) {
 
         NotificationRequestDTO notificationRequestDTO = new NotificationRequestDTO();
         notificationRequestDTO.setFiscalCode(installmentDTO.getDebtor().getFiscalCode());
         notificationRequestDTO.setOrgId(orgId);
         notificationRequestDTO.setDebtPositionTypeOrgId(debtPositionTypeOrgId);
         if (ioNotificationDTO.getIoTemplateSubject() != null && ioNotificationDTO.getIoTemplateMessage() != null && ioNotificationDTO.getServiceId() != null) {
             notificationRequestDTO.setSubject(ioNotificationDTO.getIoTemplateSubject());
-            notificationRequestDTO.setMarkdown(ioNotificationDTO.getIoTemplateMessage());
+            notificationRequestDTO.setMarkdown(replacePlaceholders(ioNotificationDTO.getIoTemplateMessage(), installmentDTO));
             notificationRequestDTO.setServiceId(ioNotificationDTO.getServiceId());
         }
         notificationRequestDTO.setAmount(installmentDTO.getAmountCents());
-        notificationRequestDTO.setOperationType(operationType);
+        notificationRequestDTO.setOperationType(NotificationRequestDTO.OperationTypeEnum.CREATE_DP);
 
         return notificationRequestDTO;
     }
+
+    private String replacePlaceholders(String markdown, InstallmentDTO installment) {
+        String placeholderDefault = "Not Specified";
```

Comment:
@robertopasetti in case of value not available, we should set an empty string or a placeholder? in what language?